contracts/raffles/src/execute.rs

```diff
@@ -146,7 +145,7 @@ pub fn execute_create_raffle(
         .plus_seconds(raffle_options.clone().raffle_duration.unwrap_or_default())
         .plus_seconds(6);
```

Comment:
I'm assuming that we add 6s to have nois respond to us on the next block after the raffle ends but I'm curious if there's anything we could do that would be a bit more sturdy in case of an extra long block. 

i'm also curious what would happen in the case that we have a chain halt at the time of a raffle's completion. we removed the determine winner and get randomness actions for the contract so would those raffles be completely stuck?